templates/macros.html.twig

```diff
@@ -1,6 +1,6 @@
 {% macro listPackages(packages, paginate, showAutoUpdateWarning, meta) %}
     <ul class="packages list-unstyled">
-        {% for package in packages|filter(package => package.replacementPackage|default('') != 'spam/spam') %}
+        {% for package in packages|filter(package => not package.isFrozen() or package.getFreezeReason().value != 'spam') %}
```

Comment:
Could instead add an isSpam() method to the enum, if you want to get rid of the magic spam string here.